cmake/modules/SwiftSource.cmake

```diff
@@ -274,6 +280,7 @@ function(_compile_swift_files dependency_target_out_var_name)
   set(module_file)
   set(module_doc_file)
 
+  set(module_command)
   if(NOT SWIFTFILE_IS_MAIN)
```

Comment:
My reasoning is that in the general flow of things, adding swift_flags -parse-as-library where you adding it, seems funky. That being said, this isn't that big of a deal.